3p/3p.js

```diff
@@ -85,7 +85,20 @@ export function writeScript(win, url, opt_cb) {
 export function loadScript(win, url, opt_cb, opt_errorCb) {
   /** @const {!Element} */
   const s = win.document.createElement('script');
-  s.src = url;
+  if (self.trustedTypes && self.trustedTypes.createPolicy) {
```

Comment:
In the description, we mention that these are treated as legacy conversions because we can't confirm nor deny that any of these inputs are safe/unsafe, and this callsite is blocking overall TT compatibility.

Can we add that in a comment somewhere here so that future visitors to this site will be able to distinguish it from a reviewed safe exemption, like some of your other PRs are?